lib/graph3d/Graph3d.js

```diff
@@ -2015,6 +2030,26 @@ Graph3d.prototype._onMouseUp = function (event) {
 };
 
 /**
+ * @param {event}  event   The event
+ */
+Graph3d.prototype._onClick = function (event) {
+  if (!this.onclick_callback)
+    return;
+  if (!this.moving) {
+    var boundingRect = this.frame.getBoundingClientRect();
+    var mouseX = getMouseX(event) - boundingRect.left;
+    var mouseY = getMouseY(event) - boundingRect.top;
+    var dataPoint = this._dataPointFromXY(mouseX, mouseY);
+    if (dataPoint)
```

Comment:
As I said before. Please use curly brackets next time to improve readability and security.